src/style.css

```
@@ -0,0 +1,51 @@
+.page__body {
+  margin: 0;
+  padding: 0;
+}
```

```suggestion
.page__body {
  margin: 0;
}
```
redundant. body hasn't default paddings.